test.sh

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env sh
-
-PORT=3001 docker-compose up --exit-code-from test "$1" test
+#!/usr/bin/env sh
+
+PORT=3001 docker-compose up --exit-code-from test "$1" test

```

Comment:
I think this file slipped in the CRLF/LF problem. Unstage it since it does not belong in this PR